packages/plugin/src/theia.d.ts

```diff
@@ -2652,7 +2652,7 @@ export module '@theia/plugin' {
          */
         static readonly Folder: ThemeIcon;
 
-        private constructor(public id: string);
+        private constructor(public id: string, public color?: ThemeColor);
```

Comment:
minor: do you mind adding the documentation here?

```suggestion
		/**
		 * Creates a reference to a theme icon.
		 * @param id id of the icon. The available icons are listed in https://code.visualstudio.com/api/references/icons-in-labels#icon-listing.
		 * @param color optional `ThemeColor` for the icon. The color is currently only used in {@link TreeItem}.
		 */
        private constructor(public id: string, public color?: ThemeColor);
```